app/views/admin/suppliers/delivery_partners/choose_cohorts.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: choose_name_admin_delivery_partners_path(continue: true))
+%>
+<h1 class="govuk-heading-l">What cohort are they working with?</h1>
```

Comment:
The copy on the wireframe is almost certainly wrong here. Double check with Shani if you like, but I think this should read "Which cohorts are \<delivery partner name\> working with?"

Reply:
I pinged Shani. In the meantime changed the copy to your suggestion.

I asked Al, and he told me that
> FYI - DfE admin journeys haven’t really been “designed”, but more scribbled on the back of an envelope in a hurry

If we are happy with the rest of the PR, that probably means it can go in (and I can update copy if / when Shani responds to me).